worlds/AutoWorld.py

```diff
@@ -121,6 +121,7 @@ class World(metaclass=AutoWorldRegister):
     A Game should have its own subclass of World in which it defines the required data structures."""
 
     option_definitions: Dict[str, Option[Any]] = {}  # link your Options mapping
+    options: Dict[str, Option[Any]]  # option names to resulting option object
```

Comment:
This gets auto-populated right?

A few lines below this, there's an example of a property with a comment "gets automatically populated".
I think that comment is important to make it clear to game implementers that they don't need to put anything in this.